packages/rocketchat-theme/client/imports/components/tabs.css

```diff
@@ -0,0 +1,27 @@
+.tabs {
+	display: flex;
+
+	width: 100%;
+
+	margin: 0 -1rem 0.5rem;
+	padding: 0 0.5rem;
+
+	border-bottom: 1px solid rgba(216,216,216, 0.2);
+}
+
+.tab {
+	margin: 0 1rem;
```

Comment:
I think you should use `transition: color .3s;` here

Reply:
👍